core/domain/exp_jobs_one_off.py

```diff
@@ -77,6 +78,82 @@
 AUDIO_FILE_PREFIX = 'audio'
 AUDIO_ENTITY_TYPE = 'exploration'
 AUDIO_DURATION_SECS_MIN_STATE_SCHEMA_VERSION = 31
+# This threshold puts a cap on the number of valid inputs, i.e.,
+# expressions/equations that can be yielded by the math expression one-off jobs.
+VALID_MATH_INPUTS_YIELD_LIMIT = 200
+
+
+def clean_math_expression(math_expression):
```

Comment:
Probably should make this private. (In general make things private where possible.)

Reply:
Done.